README.md

```diff
@@ -78,6 +78,21 @@ Deployment on [Linode](https://www.linode.com/) is performed via StackScripts, w
 
 Please submit an [issue](https://github.com/simplex-chat/simplexmq/issues) if any problems occur.
 
+[<img alt="linode" src="./img/digitalocean.png" align="right" width="300">](https://cloud.linode.com/stackscripts/748014)
+
+## Deploy SMP server on DigitalOcean
+
+[DigitalOcean](https://cloud.digitalocean.com) operates on the concept of Droplets. Droplet is a unit of deployment instantiated based on a Snapshot. You can deploy SMP server using provided [SimpleX Server Snapshot](https://marketplace.digitalocean.com/apps/simplex-server):
```

Comment:
I think this whole paragraph can be replaced with:

You can deploy SMP server using [SimpleX Server 1-click app] from DigitalOcean marketplace.

Reply:
ok